docs/content/tutorials/nginx-ingress-linkerd.md

```diff
@@ -0,0 +1,128 @@
+---
+title: NGINX Ingress Controller and Linkerd
+description: |
+  Use NGINX Ingress Controller with Linkerd
+weight: 1800
+doctypes: ["concept"]
+toc: true
+docs: "DOCS-1232"
+---
+
+# NGINX Ingress controller with Linkerd
+
+This document will walk you through how to install NGINX Ingress controller with Linkerd. Linkerd works with both NGINX Ingress controller open source and NGINX Ingress controller plus version.
+
+We are going to install the Linkderd sidecar proxy with NGINX Ingress controller.
+
+We need to install the `Linkerd` control plane utility.
+Linkerd has a great document on how to install the linkerd control plane utility on their website.
+You can find the install steps on `Linkerd` website: [Linkerd Service mesh](https://linkerd.io/2.13/getting-started/)
+
+Once you have the `Linkerd` control plane utility installed, you will need to inject the NGINX Ingress controller deployment and your application with the `Linkerd` sidecar.
+
+We are going to use the NGINX Ingress controller custom resource definitions (CRDs) and take advantage of the advanced capabilites they provide.
+
+We can install NGINX Ingress controller two ways: manfiets and with helm.
+
+Using manifests:
+You can install NGINX Ingress controller using our manifests from our documentation website.
+[NGINX Ingress controller manifest installation](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/)
+
+
+Once you have installed NGINX Ingress controller, you will need to inject the deployment with the `Linkerd` sidecar.
+This is accomplished by using the `linkerd` control plane utility.
+
+Here is an example:
+
+```bash
+kubectl get deployment -n nginx-ingress nginx-ingress -oyaml | linkerd inject - | kubectl apply -f
+```
+
+In the above example, my `NGINX Ingress controller` deployment is in the `nginx-ingress` namespace. Adjust accordingly to your environment.
+
+
+If you are using `Helm`, you can inject the sidecar in two ways.
+
+1. You can annotation your `Helm` depoloyment with the following annotation:
```

Comment:
```suggestion
Option 1. Annotation your `Helm` deployment with the following annotation:
```

Adding the the word "Option" here. When reading this someone might think this is Step 1 in doing something, and they might also do the 2nd option no realising that

Reply:
Added suggestions to doc.